src/types/common.ts

```diff
@@ -58,7 +62,10 @@ export type WriteParameters<Input> = WithSigner<
   Required<ReadParameters<Input>>
 >;
 
-export type AoMessageResult = { id: string };
+export type AoMessageResult = {
```

Comment:
Might be worth making a type of TurboAoMessageResult that uses this as an intersection

Reply:
db2592a